Verilog/README.md

```diff
@@ -0,0 +1,26 @@
+# Hello World in Verilog
+## Installing Iverilog
+
+### Ubuntu/Linux
+
+* Update the local repository cache
+
+        $ sudo apt-get update
+
+* Install (update) the Icarus Verilog package
+
+        $ sudo apt-get install verilog
+
+### Windows
+* [Download the windows installer](http://bleyer.org/icarus/) for your system.
+* Install and set the path environment variable to the bin folder of the iverilog installation directory. 
+
+
+## Running the Hello World program
+* Go to the location of your program and compile it by
+
+        iverilog HelloWorld.v -o hello.vvp
+
+* Run the compiled file by
+
+         vvp hello.vvp
```

Comment:
Enclose the code in back ticks(`). Check out this [link](https://help.github.com/articles/creating-and-highlighting-code-blocks/) for more information!